File: Triatlon-CS/Model/Participant.cs
using System;

namespace Triatlon_CSharp.Domain
{
    [Serializable]
    public class Participant : Entity<int>
    {
        public Participant() {}
        
        public Participant(string firstName, string lastName)
        {
            FirstName = firstName;
            LastName = lastName;
        }

        public String FirstName { get; set; }

        public String LastName { get; set; }

        public override string ToString()
        {
            return string.Format("Id = {0} | {1} {2}", Id, FirstName, LastName);
        }
    }
}

File: Triatlon-Java/ChatModel/src/main/java/triathlon/model/Participant.java
package triathlon.model;

import java.io.Serializable;

public class Participant extends Entity<Integer> implements Identifiable<Integer>, Serializable {
    private String firstName;
    private String lastName;

    public Participant(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }

    public Participant() {}

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    @Override
    public Integer getId() {
        return super.getId();
    }

    @Override
    public void setId(Integer integer) {
        super.setId(integer);
    }

    @Override
    public String toString() {
        return "Id = " + getId() + " " + firstName + " " + lastName;
    }
}
